doc/source/packaging.rst

```diff
@@ -132,7 +87,7 @@ the command would be;
 
 .. code-block:: bash
 
-   git tag -a 0.12.0 -m "0.12.0 release"
+   git tag -a 0.7.1 -m "0.7.1 release"
```

Comment:
Our current tag messages are on the lines of - "OpenGHG release v0.7.1" so we should make we stick to that form for this. Checked using `$git tag -n`

```suggestion
   git tag -a 0.7.1 -m "OpenGHG release v0.7.1"
```